components/Buttons/src/MDCFloatingButton+Animation.m

```diff
@@ -89,110 +86,190 @@ + (float)fab_dragCoefficient {
 }
 #endif
 
++ (CFTimeInterval)fab_animationDurationFromWallTimeDuration:(CFTimeInterval)duration
+                                                   forLayer:(CALayer *)layer {
+  CFTimeInterval finalDuration = duration;
+
+#if TARGET_IPHONE_SIMULATOR
+  finalDuration *= [self fab_dragCoefficient];
+#endif
+
+  return [layer convertTime:finalDuration fromLayer:nil];
```

Comment:
What is the convertTime being used for here?

Reply:
for the animation duration